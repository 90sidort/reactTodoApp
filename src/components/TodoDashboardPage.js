import React from 'react';
import ConnectedTodoList from './TodoList'
import TodoListFilters from '../components/TodoListFilters'

const TodoDashboardPage = () => (
    <div>
        This is my TodoDashboardPage.
        <TodoListFilters/>
        <ConnectedTodoList/>
    </div>
)

export default TodoDashboardPage