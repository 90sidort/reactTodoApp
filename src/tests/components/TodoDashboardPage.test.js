import TodoDashboardPage from '../../components/TodoDashboardPage'
import React from 'react'
import { shallow } from 'enzyme'

test('Should correctly render TodoDashboardPage component', () => {
    const wrapper = shallow(<TodoDashboardPage />)
    expect(wrapper).toMatchSnapshot()
})