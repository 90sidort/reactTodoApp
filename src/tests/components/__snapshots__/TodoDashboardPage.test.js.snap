// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should correctly render TodoDashboardPage component 1`] = `
<div>
  This is my TodoDashboardPage.
  <Connect(TodoListFilters) />
  <Connect(TodoList) />
</div>
`;
